frontend/src/app/modules/bim/ifc_models/xeokit/xeokit-server.ts

```diff
@@ -0,0 +1,83 @@
+// @ts-ignore
+import {utils} from "@xeokit/xeokit-sdk/src/viewer/scene/utils";
+import {PathHelperService} from "../../../common/path-helper/path-helper.service";
+import {GonType} from "../../../common/gon/gon.service";
+import {IFCGonDefinition} from "../pages/viewer/ifc-models-data.service";
+
+interface Window {
```

Comment:
The same, because this interface is already defined in the GonService, it seems that it shouldn't be needed here. I have tried removing it together with the one from xeokit-server.d.ts and it seems to work. Maybe this morning we missed something.